docs/data/material/getting-started/templates/sign-in-side/SignInCard.js

```diff
@@ -66,18 +74,14 @@ export default function SignInCard() {
   };
 
   return (
-    <Card
+    <StyledCard
```

Comment:
Why not get all of these other styles within the `sx` prop into the styled version?

Reply:
Responsive values as width and padding didn't work on styled version—so it felt odd to have the properties separated. Is it something I'm missing?

Reply:
The syntax for writing media queries on the styled function is a bit different (and more verbose, yes) than the `sx` prop. You can target breakpoints this way:

```
    padding: theme.spacing(1),
    [theme.breakpoints.up('sm')]: {
      padding: theme.spacing(4),
    },
```    

The first value there is applied to everything below sm. Then, the second value is applied to everything above sm.

Reply:
Is it really worth it, though?

Reply:
If we're creating a styled component and just putting box-shadows there, that doesn't feel worth it. We'd be better off with everything within the `sx` prop. All of these other inner-component properties such as display, flex, padding, etc., could also be in the styled component. And then, the `sx` prop would used more for little layout adjustments. 

That would clean up the markup more and be a more consistent overall approach imo